scripts/inner/kubestellar-kube-bind

```diff
@@ -129,6 +170,8 @@ metadata:
 data: {}
 EOF
 
+rm $KUBECONFIG
```

Comment:
use trap?

Reply:
Another deliberate choice for debuggability.